dev/integration_tests/tooltip_scroll_test/web/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
An integration test isn't needed for this change, a simple unit test can verify that the scroll view can scroll under the tooltip.